.github/workflows/ci-asp-app.yaml

```diff
@@ -0,0 +1,281 @@
+
+name: CI-ASP
+
+# ----------------------------------------------------------------------------------------------------------
+# CI Pipeline for building ASP app and deplyoing it to Azure App Service
+# - get config file via Rattus https://github.com/Saritasa/rattus
+# - deploy to Azure App Service
+# - notifies of success/failure to slack channel
+# 
+# Original Author: Rostislav Udaltsov
+# Created: 05/11/2023
+# ----------------------------------------------------------------------------------------------------------
+
+on:
+  workflow_call:
+    inputs:
+      # Build configuration
+      environment:
+        required: true
+        type: string
+        description: 'Deployment environment'
+      configuration:
+        required: true
+        type: string
+        description: 'Configuration of build Debug/Release'
+      platform_identifier:
+        required: true
+        type: string
+        description: 'Platform for build'
+      build_path:
+        required: true
+        type: string
+        description: 'Path to the application build directory'
+      template_path:
+        required: true
+        type: string
+        description: 'The path to the configuration file template used by the rattus to generate the configuration file'
+      build_folder:
+        required: true
+        type: string
+        description: 'The name of the folder where the application is going to'
+      path_to_solve_file:
+        required: true
+        type: string
+        description: 'Path to the dependency file'
+      csproj_file:
+        required: true
+        type: string
+        description: 'The path to the application build file'
+      app_name:
+        required: true
+        type: string
+        description: 'Name of the application being built'
+      azure_vault:
+        required: true
+        type: string
+        description: 'The URL of the secret repository in Azure Cloud is used by rattus to generate a configuration file'
+      deployment_slot:
+        required: true
+        type: string
+        description: 'Deployment slot in Azure Cloud'
+      runner_image:
+        required: true
+        type: string
+        description: 'The image of the runner OS on which workflow is running'
+    # For get slack channel id use this instruction
+    # https://www.wikihow.com/Find-a-Channel-ID-on-Slack-on-PC-or-Mac
+      slack_channel_id:
```

Comment:
use https://github.com/marketplace/actions/slack-send as random slack channel ID is impossible to test and hard to find the name by ID.

get the slack webhook URL
https://api.slack.com/apps/A01LM626QTZ/incoming-webhooks

Reply:
add a link above in the docs - so we can quickly openit.

Reply:
actually disregard it - as we're going to keep the webhook in the secret - it won't be readable at the code level anyway

Reply:
![image](https://github.com/saritasa-nest/saritasa-github-actions/assets/114897334/d170b162-e4fc-4c47-b788-728f0ab710e4)

done